### ISSUE TITLE
Corrupt depecy 

### ISSUE THREAD

**Issue by SirmaXX** [lucamug/style-framework#3]
Here is my plan:
  
  Add:
    elm/regex                  1.0.0
    elm/svg                    1.0.1
    lucamug/style-framework    1.0.0
    mdgriffith/elm-ui          1.1.0

Would you like me to update your elm.json accordingly? [Y/n]: Y
Dependencies loaded from local cache.
Dependencies ready!                
-- CORRUPT DEPENDENCY ----------------------------------------------------------

I ran into a problem while building the following package:

    lucamug/style-framework 1.0.0

This probably means the downloaded files got corrupted somehow.

how can ı solve this thing

**Comment by lucamug** [comment]
Hi @SirmaXX,

I tried creating a new project from scratch and everything is working fine to me. These are my steps. From a new folder:
```
$ elm init
$ elm install lucamug/style-framework
$ elm install mdgriffith/elm-ui
$ elm reactor
```
Then create a file `Main.elm` in the `src` folder with:
```
module Main exposing (main)

import Element exposing (layout)
import Framework.Button as Button
import Framework.Modifier exposing (Modifier(..))
import Html


main : Html.Html a
main =
    layout [] <| Button.button [ Medium, Success, Outlined ] Nothing "Button"
```
Then browse to http://localhost:8000/src/Main.elm and you should se a button.
This is elm.json:
```
{
    "type": "application",
    "source-directories": [
        "src"
    ],
    "elm-version": "0.19.0",
    "dependencies": {
        "direct": {
            "elm/browser": "1.0.1",
            "elm/core": "1.0.2",
            "elm/html": "1.0.0",
            "lucamug/style-framework": "1.0.0",
            "mdgriffith/elm-ui": "1.1.0"
        },
        "indirect": {
            "elm/json": "1.1.3",
            "elm/regex": "1.0.0",
            "elm/svg": "1.0.1",
            "elm/time": "1.0.0",
            "elm/url": "1.0.0",
            "elm/virtual-dom": "1.0.2"
        }
    },
    "test-dependencies": {
        "direct": {},
        "indirect": {}
    }
}
```
Let me know if you are having some issue.

**Comment by lucamug** [comment]
This should be fixed with the new `1.0.1` release: https://ellie-app.com/697qv37JqGYa1
